### PR TITLE
Added OpenCms and CKAN

### PR DESCRIPTION
**Pull request by seralf**
Added OpenCms support. This can be improved.

### CODE REVIEW

**Comment by seralf** [comment]
Hi what about these addictions? if there are still things to fix I can fix them, please let me know

File: share/apps.json [hunk 1]
```diff
@@ -620,6 +620,15 @@
 			"script": "scripts\\.chitika\\.net/",
 			"env": "ch_c(?:lient|olor_site_link)"
 		},
+		"Ckan": {
+			"website": "http://ckan.org/",
```
Comment (ElbertF) [inline share/apps.json]: Please remove the protocol from the URL, i.e. `ckan.org`.

File: share/apps.json [hunk 2]
```diff
@@ -620,6 +620,15 @@
...
+			"cats": [ 1 ],
+			"headers": { 
+				"Access-Control-Allow-Headers": "X-CKAN-API-KEY",
+                "Link": "<http://ckan.org/>; rel=shortlink" 
+            },
+            "implies": [ "python", "solr", "java", "postgres\\;confidence:80" ]
```
Comment (ElbertF) [inline share/apps.json]: Please match the case of the implies applications, i.e. `Python` instead of `python`.

File: share/apps.json (hunk 1 above)
Comment (seralf) [inline share/apps.json]: ok fixed!

**Comment by seralf** [comment]
I have fixed both the things, and added references to Solr/Lucene as libraries used. At the moment I added them under the category "databases", maybe a search frameworks or a similar category could be added later, to be more precise.

File: share/apps.json (hunk 1 above)
Comment (ElbertF) [inline share/apps.json]: It's not fixed yet. :)

Comment (seralf) [inline share/apps.json]: wow this is strange! I'll check why the push doesn't work, sorry

2015-01-26 23:43 GMT+01:00 Elbert Alias <notifications@github.com>:

Comment (seralf) [inline share/apps.json]: ah ok, I modified the Link, not the url... :-)


2015-01-26 23:44 GMT+01:00 Alfredo Serafini <seralf@gmail.com>:

File: share/apps.json [hunk 3]
```diff
@@ -620,6 +620,15 @@
 			"script": "scripts\\.chitika\\.net/",
 			"env": "ch_c(?:lient|olor_site_link)"
 		},
+		"Ckan": {
+			"website": "ckan.org/",
+			"cats": [ 1 ],
+			"headers": { 
+				"Access-Control-Allow-Headers": "X-CKAN-API-KEY",
+                "Link": "<ckan.org/>; rel=shortlink"
```
Comment (ElbertF) [inline share/apps.json]: Will this still match now that you changed the link? I'm guessing it won't.

Comment (seralf) [inline share/apps.json]: ahh ok, I have to mantai the scheme here, sorry :-D

2015-01-26 23:47 GMT+01:00 Elbert Alias <notifications@github.com>:

**Comment by seralf** [comment]
ok, from errors to typos to errors again, It should be ok now :-)

**Comment by ElbertF** [comment]
Cheers!

**Comment by seralf** [comment]
:-)

**Pull request merged**